src/control/tools/CoordinateSystemHandler.cpp

```diff
@@ -66,7 +66,7 @@ void CoordinateSystemHandler::drawShape(Point& c, const PositionInputData& pos)
 		if ( !this->modControl )	//draw out from starting point
 		{			
 			stroke->addPoint(p1);		
```

Comment:
This line has bad formatting. There seem to be many whitespaces at the end of the line that should not be there.